spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -186,25 +186,6 @@
       end
 
       describe "search" do
-        context "by display_name" do
```

Comment:
Nix'ed this since we don't actually search by display_name anymore.

Curious that it wasn't failing though